frontend/src/metabase/dashboard/actions/navigation.js

```diff
@@ -80,15 +80,18 @@ export const navigateToNewCardFromDashboard = createThunkAction(
         previousQuestion.query(),
       );
 
-      const isDrillingFromNativeModel =
-        previousQuestion.isDataset() && isPreviousNative;
+      const isFromModelOrMetric =
+        previousQuestion.type() === "model" ||
+        previousQuestion.type() === "metric";
+      const isDrillingFromNativeModelOrMetric =
+        isFromModelOrMetric && isPreviousNative;
```

Comment:
I don't know the context here, but why do we have to include metrics? Wasn't this check concerned literally only about the native models before?

Reply:
Some behaviors will stay only for models, some behaviors will apply to both metrics and models.

Since both metrics and models cannot be edited in chill mode (i.e. instead they get wrapped in ad-hoc questions; if someone wants to edit model or a metric they need to go to "edit query definition" instead) I figured drills is one of those places where the behavior for models and metrics will be the same.